listenbrainz/webserver/views/test/test_user.py

```diff
@@ -4,6 +4,12 @@
 from listenbrainz.db.testing import DatabaseTestCase
 from listenbrainz.webserver.testing import ServerTestCase
 
+from listenbrainz.listenstore.tests.util import create_test_data_for_influxlistenstore
```

Comment:
While you're adding imports, could you please seperate the `import x` and `from x import y` statements and alphabetize them?